scripts/semconv/templates/semantic_attributes.j2

```diff
@@ -27,19 +27,21 @@
     {%- endif -%}
 {%- endmacro -%}
 
-{%- macro attribute_brief(attribute) -%}
-"""
+{%- macro attribute_brief(attribute, prefix) -%}
+{%- if not prefix %}"""
+{%- endif %}
     {%- if attribute | is_deprecated %}
-Deprecated: {{ common.to_docstring(attribute.deprecated) }}.
+{{ prefix }}Deprecated: {{ common.to_docstring(attribute.deprecated) }}.
```

Comment:
I'm not too familiar with jinja. What does this change do?

Reply:
The issue here is that we comment a variable with `# MYVARIBLE: Final = 42` then pylint will warn that the following multiline comment (`""" Deprecated: ... """`) is not used. To avoid that we are adding any eventual following comment added to the variable as inline comment ("# Hello") so pylint is happy